OutOfSchool/OutOfSchool.WebApi/appsettings.json

```diff
@@ -177,7 +177,9 @@
       }
     }
   },
-
+  "ImageStorage": {
+    "BaseImageUrl": "https://minio.pozashkillia-test.iea.gov.ua/outofschool/"
```

Comment:
here's the other way around. This value should be only for dev and kubernetes environment. So it is not accidently used in prod if we forget to change. Keep this as empty string